modules/test/helpers/configmap.go

```diff
@@ -44,7 +56,7 @@ func (tc *TestHelper) ListConfigMaps(namespace string) *corev1.ConfigMapList {
 	return cms
 }
 
-// CreateEmptyConfigMap -
+// CreateEmptyConfigMap - Deprecated and unused
```

Comment:
Yepp, I think I transformed the last usage of this out of nova-operator recently. Lets delete it